free-2.md

```diff
@@ -58,12 +75,20 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think that the project would be split between language design and systems work. It would entail the construction of a formula that takes many different parameters and can simulate behavior over time. There would ideally be a non-trivial graphical display portion of the project. The language would need to allow for the creation of new variables and parameters. The DSL could be made more complex if it gave users more options (i.e. the ability to choose different variables to graph against one another). 
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is a big idea and an ambitious project, though I think that it's achievable in the timeframe of the class. It involves a lot of different UI and language design components and has the potential to be a very useful tool.
```

Comment:
I agree that it's more reasonable given the timeframe we have for class. Although I think a lot of the UI components might be a bit unrelated to the course, the language design components do show potential for a great DSL project.